### ISSUE TITLE
About experiment

### ISSUE THREAD

**Issue by Yunlong-li** [jhuang81/weak-sup-visual-grounding#1]
I could't get the experimental results in the paper when using region features and object classfication logits **extracted by myself.**
So could you please provide your features files，ths！

**Comment by drigoni** [comment]
up
